communication/README.md

```diff
@@ -36,6 +36,16 @@ devoted to the Kubernetes project. Please check the guidelines and any relevant
 chat/conversation history before posting. Spam and sales pitches are not tolerated
 on these platforms.
 
+### Appropriate Content for Community Resources
+
+All communications properties are under the [Kubernetes code of conduct]. 
+Additionally, these resources are for the contributors and users of Kubernetes, commercial usage of these properties is heavily moderated.
```

Comment:
Small nit
```suggestion
Additionally, these resources are for the contributors and users of Kubernetes; commercial usage of these properties is heavily moderated.
```

Reply:
Yeah I left it generic here because each property is different but each property page has a procedure for moderation, I wanted to leave it generic here because then we'd have to maintain that in two places. I'll add a note on how to "appeal"